system/Database/BaseBuilder.php

```diff
@@ -1090,7 +1090,7 @@ public function orNotHavingLike($field, string $match = '', string $side = 'both
      * @used-by notHavingLike()
      * @used-by orNotHavingLike()
      *
-     * @param array|RawSql|string $field
+     * @param list<int|string>|RawSql|string $field
```

Comment:
This is incorrect. See https://codeigniter4.github.io/CodeIgniter4/database/query_builder.html#id5

Like this?
```suggestion
     * @param array<string, string>|RawSql|string $field
```